thehive4py/endpoints/cortex.py

```diff
@@ -27,3 +27,26 @@ def create_responder_action(
                 "responderId": responder_id,
             },
         )
+
+    def list_analyzers(self) -> dict:
+        return self._session.make_request("GET", path="/api/connector/cortex/analyzer")
+
+    def list_analyzers_by_type(self, data_type: str) -> dict:
```

Comment:
Based on the above findings and suggestion please revisit all these methods too and align them with the docs if they are not aligned already.